src/app/modules/shared-components/components/page-navigator/page-navigator.component.html

```diff
@@ -1,6 +1,11 @@
 <div class="page-nav">
+  <alg-watch-button
+    [route]="route"
+    [startWatchGroupInfo]="startWatchGroupInfo"
+    *ngIf="allowWatching && startWatchGroupInfo"
+  ></alg-watch-button>
   <span
-    *ngIf="allowWatching"
```

Comment:
What is it? Is it the button to stop watching? The button to start watching is a specific component while the one to stop is different is managed by the parent? I don't get it.

Reply:
The problem is - we still need to support start watching user, as our new `watch button` component doesn't support user type yet.

I remember you've wrote that probably we will move content type of user into user info as group info. So then we can remove parent component params of watching feature and leave just `startWatchInfo` for start watch button component.

Reply:
Fixed